ThingIFSDK/ThingIFSDK/Gateway/GatewayAPIBuilder.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  GatewayAPIBuilder.swift
+//  ThingIFSDK
+//
+//  Copyright (c) 2016 Kii. All rights reserved.
+//
+
+import Foundation
+
+public class GatewayAPIBuilder {
+
+    private var tag: String?
+    private let app: App
+    private let gatewayAddress: NSURL
+    public var accessToken: String?
+
+    /** Set tag to this GatewayAPI instance.
+     tag is used to distinguish storage area of instance.
+     <br>
+     If the api instance is tagged with same string, It will be overwritten.
+     <br>
+     If the api instance is tagged with different string, Different key is used to store the instance.
+     <br>
+     <br>
+     Please refer to {@link GatewayAPI#loadFromStoredInstance(Context, String)} as well.
+
+     - Parameter tag: If null or empty String us passed, it will be ignored.
+     - Returns: builder instance for chaining call.
+     */
+    public func setTag(tag: String?) -> GatewayAPIBuilder
+    {
+        self.tag = tag
+        return self
+    }
+
```

Comment:
@naoki-akimoto accessToken property should be internal. Please change that.

@yongpingchen Application programmer may want to get access token. So we define `getAccessToken` as public. But We should forbid to set access token to target from application programmers. So we should not define `setAccessToken` as public.

If we make accessToken property as internal, We can set access token to the accessToken property inside of this SDK. So I think, `setAccessToken` method is not necessary.

More to say, Only getter style is same as thing-if AndroidSDK.

What do you think?